pilot/pkg/networking/plugin/authz/test_helper.go

```diff
@@ -189,3 +190,73 @@ func generatePolicyWithHTTPMethodAndGroupClaim(methodName, claimName string) *po
 		}},
 	}
 }
+
+// nolint:deadcode
+func generateExpectRBACForSinglePolicy(serviceRoleID string, rbacPolicy *policy.Policy) *policy.RBAC {
+	return &policy.RBAC{
+		Action: policy.RBAC_ALLOW,
+		Policies: map[string]*policy.Policy{
+			"service-role-" + serviceRoleID: rbacPolicy,
+		},
+	}
+}
+
+// nolint:deadcode
+// serviceRoleOrBindingID: A number represent ServiceRole or ServiceRoleBinding number.
+func generateSimpleServiceRoleBindingForNotRule(serviceRoleOrBindingID string) model.Config {
```

Comment:
This is not specific to "notRule", correct? It seems like a general binding to bind "group*" subject to any rule?

Reply:
That's correct. Will the name generateSimpleServiceRoleBindingAllGroups work or do you have any suggestions?

Reply:
Sounds good.